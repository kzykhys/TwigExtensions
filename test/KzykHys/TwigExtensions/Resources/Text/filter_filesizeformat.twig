{{ 1024|filesizeformat }}
{{ 1024|filesizeformat(true) }}
{{ 1|filesizeformat }}
{{ 2|filesizeformat }}
{{ (1152921504606850000*1048576*1024)|filesizeformat }}