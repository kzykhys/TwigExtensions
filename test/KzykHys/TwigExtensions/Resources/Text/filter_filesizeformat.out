1.0 KB
1.0 KiB
1 Byte
2 Bytes
1237.9 YB